### PR TITLE
Containerizing proposer

### PR DESCRIPTION
**Pull request by signorecello**
This PR proposes that the proposer (pun intended) is containerized so all its paths match with our deployment repo. A new script is born that simply runs this new `docker-compose` file

### CODE REVIEW

**Review by druiz0992 — CHANGES_REQUESTED** [review]
proposer.Dockerfile is based on node:14. Please set it to node:16

File: start-apps [hunk 1]
```diff
@@ -0,0 +1,2 @@
+#! /bin/bash
+docker-compose -f docker-compose.apps.yml up proposer
```
Comment (israelboudoux) [inline start-apps:2]: Hey @signorecello, what about the Challenger?

Comment (druiz0992) [reply start-apps:2]: i think we can add a separate PR for challenger. Proposer is urgent

**Review by druiz0992 — APPROVED** [review]

**Review by israelboudoux — APPROVED** [review]

File: docker-compose.apps.yml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+# Use this script for making nightfall_3 use stubs.
+services:
+
+  proposer:
+    build:
+      dockerfile: proposer.Dockerfile
+      context: .
+    ports:
+      # to use with postman and etc
+      - 8083:8080
```
Comment (druiz0992) [inline docker-compose.apps.yml:11]: There seems to be a port inconsistency with `app.mjs` where port is 8092

**Review by druiz0992 — CHANGES_REQUESTED** [review]
I launched proposer and was not able to do `curl http://localhost:8092/healthcheck`

**Comment by druiz0992** [comment]
it would be quite useful to add a proposer test that launches containers checks healthcheck, and exits. This way, we always guarantee proposer is working

**Comment by signorecello** [comment]
> 

Hey I couldn't agree more, but I think it should be part of another ticket, as some team members need this container to work ASAP

**Comment by signorecello** [comment]
Added #934 and will work on that ASAP

**Review by druiz0992 — APPROVED** [review]

**Pull request merged**